### ISSUE TITLE
Title bar glitches [new chrome]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [classilla/tenfourfox#16]
```
The title bar colour doesn't match without the tab bar.

https://bugzilla.mozilla.org/show_bug.cgi?id=618770
https://bugzilla.mozilla.org/show_bug.cgi?id=616232

We particularly need to watch for the call in M616232 c7 -- we should just stub 
that out and WONTFIX it for 10.4.

The most recent internal builds for 10.4Fx have an occasional flash of the 
window titlebar line. I'm not sure why. It may be troublesome to fix while this 
code is broken.
```

Original issue reported on code.google.com by `classi...@floodgap.com` on 14 Dec 2010 at 5:48
- Blocked on: #89
